src/ontology/pato-edit.obo

```diff
@@ -21847,6 +21847,15 @@ def: "A degree of pigmentation quality that is diversified with patches, or irre
 is_a: PATO:0002248 ! pigmented
 created_by: https://orcid.org/0000-0002-3528-5267
 
+[Term]
+id: PATO:0070010
+name: stellate morphology
+def: "A cell morphology that inheres in neurons which have dendritic processes radiating for the cell body that forms a star-like shape." [ISBN:9780123973481]
```

Comment:
> def: "A cell morphology that inheres in neurons which have dendritic processes radiating for the cell body that forms a star-like shape." [ISBN:9780123973481]

->
def: "A cell morphology that inheres in neurons that have dendritic processes radiating from the cell body forming a star-like shape." [ISBN:9780123973481]